backend/src/middleware/notifications/notificationsMiddleware.emails.spec.ts

```diff
@@ -12,6 +12,7 @@ import Factory, { cleanDatabase } from '@db/factories'
 import { createGroupMutation } from '@graphql/queries/createGroupMutation'
 import { joinGroupMutation } from '@graphql/queries/joinGroupMutation'
 import CONFIG from '@src/config'
+import { TEST_CONFIG } from '@src/config/test-mock'
 import createServer, { getContext } from '@src/server'
 
 CONFIG.CATEGORIES_ACTIVE = false
```

Comment:
TEST_CONFIG?

Reply:
You mean you would prefer to have the file named '@src/config/test-config'?